apps/emqx_dashboard/src/emqx_dashboard_app.erl

```diff
@@ -31,6 +31,7 @@ start(_StartType, _StartArgs) ->
     case emqx_dashboard:start_listeners() of
         ok ->
             emqx_dashboard_cli:load(),
+            ok = emqx_dashboard_admin:add_bootstrap_user(),
```

Comment:
There is a potential badmatch crash error.

Would it be better to prompt the user that the bootstrap_user file is incorrectly formatted and stop starting emqx, rather than Incomprehensible crashing here?